src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Color[] colors = Color.values();
+    private int randomIndex = new Random().nextInt(colors.length);
+
     public String getRandomColor() {
-        return null;
+        return colors[randomIndex].toString();
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants

Reply:
not fixed

Reply:
not fixed